apps/demos/Demos/Stepper/Overview/Angular/app/app.component.html

```diff
@@ -0,0 +1,123 @@
+<div class="stepper-demo">
+  <div class="widget-container">
+    <div class="widget-wrapper" [ngClass]="'widget-wrapper-' + orientation">
+      <div class="stepper-wrapper">
+        <div id="iconsLabel" class="stepper-label">Icons and Labels</div>
+        <dx-stepper
+          id="icons"
+          [elementAttr]="{ 'aria-labelledby': 'iconsLabel' }"
+          [selectedIndex]="2"
+          [orientation]="orientation"
+          [linear]="navigationMode"
+          [selectOnFocus]="selectOnFocus"
+          [rtlEnabled]="rtlMode"
+        >
+          <dxi-item
```

Comment:
Here and in other examples. Let's use *ngFor construction, this way it becomes more readable

```suggestion
 <dxi-item
    *ngFor="let step of steps"
    [label]="step.label"
    [icon]="step.icon"
    [text]="step.text"
    [optional]="step.optional"
  ></dxi-item>
```